CarbStor_Community_Models/Community_Model_Information.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
```

Comment:
This `# In[ ]:` should be removed throughout the script, they are artifacts from saving a jupyter notebook to python script and are just unnecessary clutter now.